test/unit/legacy_unit/egg/exit.r

```diff
@@ -1,7 +0,0 @@
-main@global(128) {
```

Comment:
imho this can be into libr/egg/t and installed as sample ragg programs in the share/ directory